### ISSUE TITLE
What value does making content bring me? (Not a typo)

### ISSUE THREAD

**Issue by davidwesst** [davidwesst/blog#16]
That's not to say the content should provide value to the reader, also known as you, but it's still a very important question to answer. 

With the reboot of my blog and getting back into social media, this has become the first question I ask myself before writing a post, or planning a livestream, or scripting a video, or even saving a link. It's selfish, but if the post isn't valuable to me, then why would it be valuable to you?

In my previous professional life, the content I created with exclusively for the readers and for Microsoft. I made content, they awarded me an MVP award with some great perks and an awesome community which I talked about here on my blog [LINK](). It was a good deal and made sense at the time. But that was then, and this is now. Now when I look at creating content, first person it needs to bring value to is me. 

## So why bother blogging at all?

I didn't. 

At least I didn't for a few years and just left my blog and website to be a development science experiment. Reason being, I didn't know the value the blog could bring me.

I'll leave the numerous self-driven arguments and half baked reasons I tried to give myself out of it, but ultimately it didn't make a lot of sense to keep on blogging.

Until it did again.

## Sharing my Self-Retrospectives

Even though I stopped blogging and stopped doing my annual self retrospectives (like this one from 2014: [LINK]() and this one from 2015 [LINK]()) the self-reflective process never stopped. Rather, my the self-retrospectives evolved into smaller chunks of thought that I would or share with trusted friends or family to get opinions on the deep thoughts from this inner monologue I maintained.

Then a pandemic started and I was unable to share, at least not with the frequency and ease that I used to.

## The Absence of Validation

It's the inability to share the way I was used to got me thinking differently. I started to realize that sharing was my way to get approval and a pat on the head for an idea. I didn't need to take action with my website or my social media presence, because I already had a bunch of people tell me it was a good idea. Why bother doing it when I already got it validated?

And so I tested my theory and stopped sharing my ideas on social media. Not long after that, I stopped sharing my ideas with my trusted friends virtually (unless I had something to show, which I never did) not because I didn't want to, but because I needed to learn to do this for myself.

_For me._

## TL;DR; (Conclusion)

This is why I ask the question: "What value does <idea> it bring to me?" before I write a post or whatever other thing I'm thinking about committing time to. If I don't see a point in it, the idea might not be bad, but it's just not for me. My time is better spent on an idea, post, or science experiment that I think is worth trying.

And if I think it's a good idea, then that's one person. Odds are that there is someone else that'll think my idea is worth investing time into as well.

---

Thanks for playing.
~ DW
